src/TwigYard/Middleware/Url/UrlMiddleware.php

```diff
@@ -95,6 +95,12 @@ public function __invoke(ServerRequestInterface $request, ResponseInterface $res
         if ($host !== $canonicalUrl) {
             return new RedirectResponse($request->getUri()->getScheme() . '://' . $canonicalUrl, 301);
         }
+
+        $ssl = isset($configs[$host]['url']['ssl']) && $configs[$host]['url']['ssl'];
```

Comment:
This could be written as: `$ssl = !empty($configs[$host]['url']['ssl']);`
It is shorter, more readable and does the same thing.